### PR TITLE
Frozen Star Background Checks

### PR DESCRIPTION
**Pull request by EvenInDeathIStillServe**
Firearm and Ammunition vending machines registered to the Frozen Star™ Corporation now conduct an extensive and scrutinous background check on all customers, in order to evade legal liability in case of selling weapons to a known criminal.

### CODE REVIEW

**Comment by SpadesSlick** [comment]
The memes are strong in this one.

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -674,6 +675,8 @@
 					src.status_error = 1
 				else
 					src.status_message = "Please swipe a card or insert cash to pay for the item."
+					if(no_criminals)
+						src.status_message += " We are legally required to ask if you are currently wanted by any law enforcement organizations. If so, please cancel the purchase, announce your location to local law enforcement and wait for them to collect you."
```
Comment (nichlas0010) [inline code/game/machinery/vending.dm]:
```suggestion
						status_message += " We are legally required to ask if you are currently wanted by any law enforcement organizations. If so, please cancel the purchase, announce your location to local law enforcement and wait for them to collect you."
```

Comment (NanakoAC) [reply code/game/machinery/vending.dm]: good change

as long as you're poking around in here @EvenInDeathIStillServe could you remove all the "src." from this file?

**Comment by EvenInDeathIStillServe** [comment]
Done. I have successfully turned a 4 line meme PR into 135 more replaced lines.

**Pull request merged**